src/net.cpp

```diff
@@ -754,7 +754,7 @@ const uint256& CNetMessage::GetMessageHash() const
 
 
 // requires LOCK(cs_vSend)
-size_t CConnman::SocketSendData(CNode *pnode)
```

Comment:
Well, writing to the peer itself here is writing to pnode, which isn't const, no?